gen/generate.go

```diff
@@ -326,24 +340,25 @@ func generateGenesisFiles(outDir string, protoVersion protocol.ConsensusVersion,
 		fmt.Fprintln(verboseOut, protoVersion, protoParams.MinBalance)
 	}
 
+	if rewardsBalance < protoParams.MinBalance {
+		// Needs to at least have min balance
+		rewardsBalance = protoParams.MinBalance
+	}
+
 	records["FeeSink"] = basics.AccountData{
 		Status:     basics.NotParticipating,
 		MicroAlgos: basics.MicroAlgos{Raw: protoParams.MinBalance},
 	}
 	records["RewardsPool"] = basics.AccountData{
 		Status:     basics.NotParticipating,
-		MicroAlgos: basics.MicroAlgos{Raw: defaultIncentivePoolBalanceAtInception},
+		MicroAlgos: basics.MicroAlgos{Raw: rewardsBalance},
 	}
 
 	sinkAcct := genesisAllocation{
-		Name:   "FeeSink",
-		Stake:  protoParams.MinBalance,
-		Online: basics.NotParticipating,
+		Name: "FeeSink",
 	}
 	poolAcct := genesisAllocation{
-		Name:   "RewardsPool",
-		Stake:  defaultIncentivePoolBalanceAtInception,
-		Online: basics.NotParticipating,
+		Name: "RewardsPool",
 	}
```

Comment:
Don't you want this to have `Stake: rewardsBalance`?

Why are you removing the Online status? I believe that changes the semantics so that they're `Offline` which is slightly different than `NotParticipating`

Reply:
@winder I attempted to address these questions in https://github.com/algorand/go-algorand/pull/4643#discussion_r1009879473.  Can you have a look at the reasoning there and let me know if it makes sense to you?

I'm happy to continue discussion in this thread though I didn't want to duplicate the rationale again.

Reply:
Resolving this and moving to the other comment.